loom/common/core/allocator.h

```diff
@@ -78,6 +78,8 @@
  *
  */
 
+#define LOOM_ALLOCATOR_ALIGN_MASK (8-1)
```

Comment:
Somewhat, I'll add a comment, but it's just a place where that number is kept, the allocator doesn't do anything special with it.